cetmix_tower_server/models/cx_tower_server.py

```diff
@@ -1386,9 +1408,7 @@ def _command_runner_ssh(
         if log_record:
             log_record.finish(
                 finish_date=fields.Datetime.now(),
-                status=command_result["status"],
-                response=command_result["response"],
-                error=command_result["error"],
+                **command_result,
```

Comment:
Nope, I meant you could use [this kwargs](https://github.com/cetmix/cetmix-tower/blob/7ecccc28fde7b8b0cc8aac8e1e1260f72376fc51/cetmix_tower_server/models/cx_tower_server.py#L1382) to pass the ssh job pid. However I see this is not the case.